src/webgpu/api/operation/rendering/draw.spec.ts

```diff
@@ -44,7 +28,330 @@ TODO:
   - mode= {draw, drawIndexed, drawIndirect, drawIndexedIndirect}
 `;
 
+import { params, pbool, poptions } from '../../../../common/framework/params_builder.js';
 import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { assert } from '../../../../common/framework/util/util.js';
 import { GPUTest } from '../../../gpu_test.js';
 
 export const g = makeTestGroup(GPUTest);
+
+g.test('arguments')
+  .desc(
+    `Test that draw arguments are passed correctly by drawing triangles in a grid.
+Horizontally across the screen are triangles with increasing "primitive id".
```

Comment:
nit: screen -> texture since we aren't using the swapchain.

Also maybe this test could be simpler if it were using points? WDYT?

Reply:
I think it's easier to test points in a test like this since we're translating from 0-1 or NDC coordinates to pixel offsets for the expectations. Triangles let us avoid getting the exact point and instead just check where the center of the triangle is.

Reply:
Ack

Reply:
The nit: about screen->texture in the comment could still be addressed. LGTM otherwise.